core/pom.xml

```diff
@@ -220,6 +220,21 @@
       <version>3.2.10</version>
     </dependency>
     <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-scala_2.10</artifactId>
+      <version>2.3.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.jersey</groupId>
+      <artifactId>jersey-server</artifactId>
+      <version>1.9</version>
```

Comment:
Since we want this dependency to track the one used by YARN (to avoid adding more dependencies), can you add `<scope>${hadoop.deps.scope}</scope>` here?

It's a not-so-explicit indication that this is the same version that hadoop uses (and helps people who don't package hadoop classes with Spark).